cltk/stem/lemma.py

```diff
@@ -3,14 +3,15 @@
 __author__ = ['Kyle P. Johnson <kyle@kyle-p-johnson.com>']
 __license__ = 'MIT License. See LICENSE.'
 
+import warnings
 from cltk.utils.cltk_logger import logger
 import importlib.machinery
 from nltk.tokenize.punkt import PunktLanguageVars
 import os
 
 AVAILABLE_LANGUAGES = ['greek', 'latin']
 
-
+# Deprecated; remove from future release?
```

Comment:
should we use: https://pypi.org/project/deprecation/
I know it seems a little overkill, but I like that it provides runtime warnings to users.
a single comment is too easy to miss for many users

Reply:
Cf. https://github.com/cltk/cltk/issues/632

Reply:
Going to resolve this thread—but we should resume it at #632.